torchtune/modules/rlhf/collate.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Dict, List, Tuple
+
+import torch
+from torch.nn.utils.rnn import pad_sequence
+
+from torchtune.data import CROSS_ENTROPY_IGNORE_IDX
+
+
+def left_padded_collate(
```

Comment:
I think this can go in generate `collate.py` - it should be general enough :)

Reply:
Would you like to see it match the signature of `padded_collate`, too? There's also a more convoluted suggestion here https://github.com/pytorch/torchtune/pull/1005#discussion_r1648391833